tasks/javascript/easy/sum.js

```diff
@@ -2,3 +2,7 @@
 
 // TODO: Write a function that returns the sum of two numbers
 
+
+a =6
+b=7
+console.log(a+b)
```

Comment:
This does not work since the variables would need to be declared either with `let` or `const`. Also, the description requests a function that returns the sum of two numbers, not just a console.log.